base_layer/wallet/src/util/emoji.rs

```diff
@@ -132,7 +132,7 @@ impl EmojiId {
         Self(id)
     }
 
-    fn byte_vec(s: &str) -> Result<Vec<u8>, ()> {
+    pub fn byte_vec(s: &str) -> Result<Vec<u8>, ()> {
```

Comment:
This was also left private for similar reasons.

Reply:
(1) I'll push a PR shortly with a suggested alternative approach that keeps these functions private

Reply:
Re-based on PR#1469